moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -262,79 +264,95 @@ export default function TheMap({
     isCreatingComponent && shouldShowDrawControls;
   const shouldShowEditDrawControls =
     isEditingComponent && shouldShowDrawControls;
+  const shouldShowZoomAlert =
+    currentZoom < MIN_SELECT_FEATURE_ZOOM &&
+    (isCreatingComponent || isEditingComponent) &&
+    isDrawing === false;
 
   return (
-    <MapGL
-      ref={mapRef}
-      initialViewState={initialViewState}
-      interactiveLayerIds={interactiveLayerIds}
-      onMouseEnter={onMouseEnter}
-      onMouseLeave={onMouseLeave}
-      onMoveEnd={onMoveEnd}
-      onClick={onClick}
-      cursor={cursor}
-      mapStyle={basemaps[basemapKey].mapStyle}
-      {...mapParameters}
-    >
-      <BasemapSpeedDial basemapKey={basemapKey} setBasemapKey={setBasemapKey} />
-      <GeocoderControl position="top-left" marker={false} />
-      {shouldShowCreateDrawControls && (
-        <CreateComponentDrawTools
-          createDispatch={createDispatch}
-          linkMode={linkMode}
-          setCursor={setCursor}
-          setIsDrawing={setIsDrawing}
+    <>
+      <MapAlertSnackbar
+        isOpen={shouldShowZoomAlert}
+        message="Zoom in to select map features"
+        severity="error"
+      />
```

Comment:
This is the important part - the rest of the diff comes from me wrapping the **MapGL** component with a fragment.